doc/radosgw/notifications.rst

```diff
@@ -161,55 +162,81 @@ name of an existing topic and different endpoint values).
+- OpaqueData: opaque data is set in the topic configuration and added to all
+  notifications that are triggered by the topic.
+- persistent: indicates whether notifications to this endpoint are persistent
+  (=asynchronous) or not persistent. (This is "false" by default.)
 
 - HTTP endpoint
 
  - URI: ``http[s]://<fqdn>[:<port]``
- - port defaults to: 80/443 for HTTP/S accordingly
- - verify-ssl: indicate whether the server certificate is validated by the client or not ("true" by default)
- - cloudevents: indicate whether the HTTP header should contain attributes according to the `S3 CloudEvents Spec`_ ("false" by default)
+ - port: defaults to "80 for HTTP or 443 for HTTPS.
+ - verify-ssl: indicates whether the server certificate is validated by the
+   client. (This is "true" by default.)
+ - cloudevents: indicates whether the HTTP header should contain attributes
+   according to the `S3 CloudEvents Spec`_. (This is "false" by default.)
 
 - AMQP0.9.1 endpoint
 
  - URI: ``amqp[s]://[<user>:<password>@]<fqdn>[:<port>][/<vhost>]``
- - user/password defaults to: guest/guest
- - user/password may only be provided over HTTPS. If not, topic creation request will be rejected.
- - port defaults to: 5672/5671 for unencrypted/SSL-encrypted connections
- - vhost defaults to: "/"
- - verify-ssl: indicate whether the server certificate is validated by the client or not ("true" by default)
- - if ``ca-location`` is provided, and secure connection is used, the specified CA will be used, instead of the default one, to authenticate the broker
- - amqp-exchange: the exchanges must exist and be able to route messages based on topics (mandatory parameter for AMQP0.9.1). Different topics pointing to the same endpoint must use the same exchange
- - amqp-ack-level: no end2end acking is required, as messages may persist in the broker before delivered into their final destination. Three ack methods exist:
-
-  - "none": message is considered "delivered" if sent to broker
-  - "broker": message is considered "delivered" if acked by broker (default)
-  - "routable": message is considered "delivered" if broker can route to a consumer
-
-.. tip:: The topic-name (see :ref:`radosgw-create-a-topic`) is used for the AMQP topic ("routing key" for a topic exchange).
+ - "user/password": defaults to "guest/guest".
+ - user/password: must be provided only over HTTPS. Topic creation requests will
+   otherwise be rejected.
+ - port: defaults to: 5672 for unencrypted connections and 5671 for
+   SSL-encrypted connections.
+ - vhost: defaults to: "/"
+ - verify-ssl: indicates whether the server certificate is validated by the
+   client. (This is "true" by default.)
+ - if ``ca-location`` is provided and secure connection is used, the specified
+   CA will be used to authenticate the broker. The default CA will not be used. 
+ - amqp-exchange: the exchanges must exist and be able to route messages based
+   on topics. (This parameter is mandatory for AMQP0.9.1). Different topics
```

Comment:
@zdover23 For consistency's sake, the period should be before the closing parenthesis.